features/toggle_sam_warning.feature

```diff
@@ -6,17 +6,27 @@ Feature: Toggling visibility on the sam status warning
   Scenario: When sam status is verified
     Given I am a user with a verified SAM account
     When I sign in and verify my account information
-    Then I should not see an alert warning me about my SAM registration
+    Then I should not see a warning about my SAM registration
 
   Scenario: When sam status unverified
     Given I am a user without a verified sam account
     When I sign in and verify my account information
-    Then I should see an alert warning me that my SAM registration is not complete
+    Then I should see a warning that my SAM registration is not complete
 
-  @wip
+  @javascript
   Scenario: Toggling the sam status warning
     Given I am a user without a verified sam account
+    And there is an open auction
     When I sign in and verify my account information
-    When I collapse the alert warning me about my SAM registrtation
-    Then I will not see the alert
-    And I will see an link to expand the alert
+    And I collapse the warning about my SAM registrtation
```

Comment:
Sweet. Do you think can start converting some of the other feature tests to Cucumber?